### DISCUSSION TITLE
How to define a time-dependent boundary condition？

### DISCUSSION THREAD

**Discussion by songjuzheng** [su2code/SU2#2054] [Q&A]
Hi all,

Can SU2 define a time-dependent boundary condition？ For example, Mach = 0.5 + 0.1 * sin(Time).

Thanks

**Comment by bigfooted** [comment]
There is an example of an unsteady wall boundary condition in the py_wrapper folder (so you have to use the python wrapper). But the current functionality is limited, I don't think you can currently use an unsteady inlet boundary condition.
